### ISSUE TITLE
Error in NASA Exoplanet Archive notebook

### ISSUE THREAD

**Issue by hickst** [WorldWideTelescope/pywwt-notebooks#37]
In Step 3: Adding a tabular data layer:
```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/usr/local/src/anaconda3/envs/wwt/lib/python3.7/site-packages/traitlets/traitlets.py in get(self, obj, cls)
    534         try:
--> 535             value = obj._trait_values[self.name]
    536         except KeyError:

KeyError: 'time_decay'

During handling of the above exception, another exception occurred:

TypeError                                 Traceback (most recent call last)
<ipython-input-6-72e5cb021a0a> in <module>()
----> 1 layer = wwt.layers.add_table_layer(exoplanets)

/usr/local/src/anaconda3/envs/wwt/lib/python3.7/site-packages/pywwt/layers.py in add_table_layer(self, table, frame, **kwargs)
    240         if table is not None:
    241             layer = TableLayer(self._parent, table=table,
--> 242                                frame=frame, **kwargs)
    243         else:
    244             # NOTE: in future we may allow different arguments such as e.g.

/usr/local/src/anaconda3/envs/wwt/lib/python3.7/site-packages/pywwt/layers.py in __init__(self, parent, table, frame, **kwargs)
    433         self._on_trait_change({'name': 'time_att', 'new': self.time_att})
    434         self._on_trait_change({'name': 'time_series', 'new': self.time_series})
--> 435         self._on_trait_change({'name': 'time_decay', 'new': self.time_decay})
    436 
    437         self._on_trait_change({'name': 'cmap', 'new': self.cmap})

/usr/local/src/anaconda3/envs/wwt/lib/python3.7/site-packages/traitlets/traitlets.py in __get__(self, obj, cls)
    573             return self
    574         else:
--> 575             return self.get(obj, cls)
    576 
    577     def set(self, obj, value):

/usr/local/src/anaconda3/envs/wwt/lib/python3.7/site-packages/traitlets/traitlets.py in get(self, obj, cls)
    536         except KeyError:
    537             # Check for a dynamic initializer.
--> 538             default = obj.trait_defaults(self.name)
    539             if default is Undefined:
    540                 warn(

/usr/local/src/anaconda3/envs/wwt/lib/python3.7/site-packages/traitlets/traitlets.py in trait_defaults(self, *names, **metadata)
   1575 
   1576         if len(names) == 1 and len(metadata) == 0:
-> 1577             return self._get_trait_default_generator(names[0])(self)
   1578 
   1579         trait_names = self.trait_names(**metadata)

TypeError: 'Quantity' object is not callable
```

**Comment by pkgw** [comment]
This is an error related to traitlets 5.0 that will be fixed when I can merge in #275.

**Comment by hickst** [comment]
This issue appears to be solved, as a consequence of the fix for this issue: https://github.com/WorldWideTelescope/pywwt/issues/272
